### ISSUE TITLE
Data Integration

### ISSUE THREAD

**Issue by VaghehDashti** [fani-lab/OpeNTF#178]
Hello @hosseinfani,
I created this issue as you asked to put updates for uploading the models and results of our temporal paper to ms-teams.
dblp is almost done. only bnn_emb is left.
imdb bnn, bnn_emb, and tbnn is done. tbnn_emb and tbnn_dt2v_emb is left.
uspt bnn and bnn_emb is done. tbnn, tbnn_emb, and tbnn_dt2v is left.

**Comment by VaghehDashti** [comment]
Hi @hosseinfani,
imdb and upst's results are uploaded as well.
I will move forward to rrn's results.

**Comment by hosseinfani** [comment]
@VaghehDashti 
you had to upload to a seprate folder not the opentf0.1! You overwrite the previous results. Let's talk about it today at lab.

**Comment by VaghehDashti** [comment]
Hi @hosseinfani,
rrn's results plus the reformulated input data and the predictions of the model are uploaded to ms-teams as well.

**Comment by hosseinfani** [comment]
@VaghehDashti 
Thank you. All is done then, right?

**Comment by VaghehDashti** [comment]
@hosseinfani 
Yes, it is done.

**Comment by hosseinfani** [comment]
@VaghehDashti 
I've the files. I think the preprocessed folder is missing. We need the pkl files in preprocessed folder on which these results have been obtained.

**Comment by VaghehDashti** [comment]
Hi @hosseinfani,
Oh, I didn't know about this. I will upload them and let you know when done.

**Comment by VaghehDashti** [comment]
Dblp's preprocessed data is uploaded. imdb and uspt are being downloaded.

**Comment by VaghehDashti** [comment]
Hi @hosseinfani,
All is done.

**Comment by hosseinfani** [comment]
@VaghehDashti 
Thank you. I download the files from msteams in my local drive also. Just to make sure the upload happened correctly, the full size is 70.5 GB, is that the right size in your local drive/sharcnet drive?

**Comment by VaghehDashti** [comment]
Hi @hosseinfani,
I don't know the exact size because after uploading them I would delete them from the local machine. However, I should let you know that my earlier guess about the total uncompressed size was wrong because there were a lot of old experiments or experiments with different filterings that I deleted before downloading from sharcnet and uploading to ms-teams.

**Comment by VaghehDashti** [comment]
Hi @hosseinfani,
I've uploaded the results of the experiments on dblp with [128,64,128] and [256,128,64,128,256] to ms-teams. The performance of bnn and bnn_emb has decreased on dblp. I think the decrease in performance is due to the size of the models getting larger while the #training instance has not increased, so the model is probably underfitting. Although, changing the model size may require a new learning rate, batch size, #epochs, and so on for each dataset to get better or same performance. I'm downloading imdb's results and will upload them as soon as they are ready. uspt with bnn is still running for [256,..,256]. I will let you know when the results on the other datasets are ready and uploaded to ms-teams.
What should be my next step if the same drop in performance emerge on the other 2 datasets?

**Comment by hosseinfani** [comment]
Hi @VaghehDashti 
Thanks. I replied in #120

**Comment by VaghehDashti** [comment]
Hi @hosseinfani,
The results of uspt is also uploaded to ms-teams for 3 and 5 layers and both models.

**Comment by VaghehDashti** [comment]
Hi @hosseinfani,
I've pushed the results of the the ecir paper to github. Please pull.

**Comment by VaghehDashti** [comment]
Hi @hosseinfani,
The results for imdb with 3 layers [128,64,128] with learning rates of 0.01 and 0.001 are uploaded to ms-teams.

**Comment by VaghehDashti** [comment]
Hi @hosseinfani,
I wanted to let you know that the data for experiments on #layers/nodes with different learning rates are uploaded to ms-teams and the results are pushed to dev. Will update here after finishing the experiments on #bayesian samples.

**Comment by VaghehDashti** [comment]
@hosseinfani 
I've uploaded the results for #bs={3,5,10} for all 3 datasets to ms-teams and pushed the results to dev as well.
